### DISCUSSION TITLE
[Feature Request] Add Option Similar to "Selected Mods" that Separates Custom Levels Within Modifiers (ex. DT 1.25 and DT 1.5) or Ranked vs Unranked

### DISCUSSION THREAD

**Discussion by OnlyJustEthan** [ppy/osu#29554] [Ideas]
Basically just the title, I play songs using lower speeds of DT and will slowly ramp up, and it would be neat to be able to see how my scores on the local leaderboard compare to each other at a level of speed without having to check what multiplier of DT the score was set at. Alternatively, having a toggle for ranked vs unranked scores in the local score list could be used for comparing growth through all play levels or being able to easily filter out the ones that were just for practice.

**Comment by Tenexxt** [comment]
Scores with custom DT and HT already have a tail next to the mod with the the speed multiplier on it in-game
![image](https://github.com/user-attachments/assets/0f8af375-6106-4a7f-8abb-57f5ecd934f4)
As for the web, its already in this post #29370

**Reply by OnlyJustEthan** [reply]
You're misunderstanding, I'm not looking to know what the multiplier is on a given play. Rather, I want to be able to show only replays that have that specific multiplier. 

Say I'm going to 1.4 multiplier after playing on 1.2 for a while. Since it's going to be harder, I will theoretically be setting lower scores and those plays will go lower on the leaderboard. Right now I would have to scroll down to see and compare the 1.4 multiplier scores which will probably have the 1.2 scores between them.

If we had a filter for the multiplier within the mod filter, that would allow only scores with the 1.4 multiplier to be shown while the 1.2 multiplier scores are hidden.

**Reply by Tenexxt** [reply]
For local scores a filter for mods would need to be added overall (since its currently lacking one), but this would be usefull for people who have a lot of scores on the same map (unlike me who has 13 scores with 160 plays on the same map lol).
This also could be implemented into online leaderboards, but im not a dev so im not going to act like i have a any sort of say about this being added or not.
Also, sorry for misunderstanding

**Reply by OnlyJustEthan** [reply]
No worries, I can't be annoyed about me not explaining super good.

As far as a general mod filter, do you not mean something like what happens with the Selected Mods checkbox above replays? It's a little roundabout because you still have to select the mod in the mod menu, but it sounds like it would do effectively the same thing. 

It is also a Lazer feature, I now hope I haven't posted on the Stable-only forum by accident.

**Reply by Tenexxt** [reply]
Stable is featured locked so it's quite impossible to request a feature for it.

As for mod select for replays, it could work the same as mod select for online leaderboards, you click on a mod, and you view scores with only that mod/only a certain combination of mods - exept you'd also could change the "type" of the mod you're viewing ( DT 1.2x, HD only fade out approach circles etc. )

**Comment by GelyLily** [comment]
I was making my own post for this which I'm going to just put here instead as it seems this is similar and I want to bump it:

---------------------------------------------------

I really like the "Selected Mods" checkbox in the local leaderboards. I've been playing a map nomod and wanted to start rate changing it 0.05x at a time, so turning on the Selected Mods filter was really handy the first time I rate changed. I then noticed as I increased the speed that the scores from the previous rate change were still showing which feels like a bit of a shame!

I guess I'm asking for the local leaderboards to treat each speed increment increase as its own mod (purely for the purpose of filtering scores).

I have attached a screenshot of what I'm doing in game here to illustrate why I feel the current implementation is lacking. I'm sure you can see how the 1.05x scores are now not relevant to me, I just want to see those 1.10x scores. If I were to continue this all the way up to double time the score list would get very long and hard to read.
![osu_2025-11-30_19-45-18](https://github.com/user-attachments/assets/1768b1f6-6b57-4892-ac56-df72f441afaf)
